spec/models/work_package/work_package_acts_as_customizable_spec.rb

```diff
@@ -202,23 +212,41 @@ def setup_custom_field(cf)
              is_for_all: true)
     end
 
+    let!(:custom_field_of_project_and_type_for_all) do
```

Comment:
The plural form would help me better understand what this custom field is about throughout the rest of the spec file below:

```suggestion
    let!(:custom_field_of_projects_and_types_for_all) do
```